lib/build_manager.py

```diff
@@ -43,9 +43,9 @@ def _build_packages(self, distro, packages):
         """
```

Comment:
Never mind my previous comment. I just noticed we do not obtain lsb_name from the system installed with the Linux distribution but from a value provided by user (via config file or command line). And this is compared against a list of constants we define.